packaging/package-all.sh

```diff
@@ -1,25 +1,28 @@
 #!/bin/bash
 set -e
 
-if [ $# -ne "2" ]; then
-    echo "Usage: `basename $0` version outputdir"
+if [ $# -ne "1" ]; then
+    echo "Usage: `basename $0` version"
     exit 1
 fi
 
+command -v python >/dev/null 2>&1 || { echo >&2 "The OpenRA mod template requires python."; exit 1; }
```

Comment:
Would you mind adding checks for the commands used by the sub-packaging scripts here too?
It will be better to fail up front than part way through.